config/packages/fos_http_cache.yaml

```diff
@@ -0,0 +1,13 @@
+fos_http_cache:
+    proxy_client:
+        symfony:
+            use_kernel_dispatcher: true
```

Comment:
think i would remove this similar to whats done in https://github.com/sulu/skeleton/pull/75/files

Reply:
This file is still there in the skeleton ... I removed that config from `config/packages/prod/sulu_http_cache.yaml`

Reply:
oh you are right 🙂